src/LabelComponent.jsx

```diff
@@ -0,0 +1,158 @@
+import { useState, useEffect } from "react";
+import "./LabelComponent.css";
+
+const LabelComponent = () => {
+  const [inputValue, setInputValue] = useState("");
+  const [labelsData, setLabelsData] = useState([]);
+  const [filteredSuggestions, setFilteredSuggestions] = useState([]);
+  const [selectedLabels, setSelectedLabels] = useState([null]);
```

Comment:
```suggestion
  const [selectedLabels, setSelectedLabels] = useState([]);
```
empty array should be sufficient as default value